Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -361,24 +361,24 @@
     <string name="submission_test_result_positive_steps_positive_heading">Testergebnis Positiv</string>
     <string name="submission_test_result_positive_steps_positive_body">Ihr Testergebnis wurde erfolgreich als “Positiv” verifiziert.</string>
     <string name="submission_test_result_positive_steps_warning_others_heading">Andere warnen</string>
-    <string name="submission_test_result_positive_steps_warning_others_body">Teilen Sie Ihre Zufallskennungen, damit andere gewarnt werden können.</string>
+    <string name="submission_test_result_positive_steps_warning_others_body">Teilen Sie Ihre Zufallskennungen, damit Andere gewarnt werden können.</string>
     <string name="submission_test_result_positive_continue_button">Weiter</string>
 
     <string name="submission_test_result_invalid_steps_invalid_heading">Test fehlerhaft</string>
     <string name="submission_test_result_invalid_steps_invalid_body">Es gab ein Problem bei der Auswertung Ihres Tests. Bitte kontaktieren Sie das Gesundheitsamt um Information zum weiteren Vorgehen zu erhalten. Bitte entfernen Sie den Test wieder aus der Corona-Warn-App, damit Sie bei Bedarf einen neuen Test hinterlegen können.</string>
     <string name="submission_test_result_invalid_remove_test_button">Test entfernen</string>
     <!-- Submission Tan -->
     <string name="submission_tan_title">TAN Eingabe</string>
-    <string name="submission_tan_body">Geben Sie bitte die TAN Nummer ihres Befundes ein. Die TAN ist 7-stellig und Groß- und Kleinschreibung muss nicht beachtet werden.</string>
+    <string name="submission_tan_body">Geben Sie bitte die TAN (Transaktionsnummer) Ihres Befundes ein. Die TAN ist 7-stellig und Groß- und Kleinschreibung muss nicht beachtet werden.</string>
     <string name="submission_tan_button_text">Weiter</string>
     <!-- Submission Intro -->
     <string name="submission_intro_title">Wurden Sie getestet?</string>
     <string name="submission_intro_headline">So funktionert das Corona-Warn-System</string>
-    <string name="submission_intro_text_a">Damit die Corona-Warn-App funktioniert, wird die freiwillige Mithilfe von allen Corona positiven Personen benötigt.</string>
+    <string name="submission_intro_text_a">Damit die Corona-Warn-App funktioniert, wird die freiwillige Mithilfe von allen Corona-positiven Personen benötigt.</string>
```

Comment:
grammatically, I think that it likely has to be lowercase "corona-positiven", however I kept the upper case spelling to add more visual guidance here.